demo/seamless_server/README.md

```diff
@@ -0,0 +1,33 @@
+## to start the server
```

Comment:
@lh5844 Could you update this README to reflect the steps you needed to get the server backend set up? I think you've had the most recent experience setting it up, so it would be helpful to incorporate anything you noticed which wasn't in the instructions. Also, we can remove the part about the google cloud account, and downloading the streaming models (since this is intended as a general-purpose demo). 

For now, you could update the README to describe how to set up the demo for the dummy S2T/S2S models, but in the future we could switch it to e.g. the Whisper tiny model for running locally